src/shared/components/Button/Button.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { LandingButton } from "./LandingButton";
+
+import "./Button.css";
+
+export const Button = ({
+  landingButton,
+  className,
+  text,
+  transparentBg,
+  handleClickButton,
+}) => {
+  if (landingButton) {
```

Comment:
Yeah I think this is so redundant. Imo, it's better to use the landingButton component directly as stated above.

Reply:
It is a bit redundant. But as stated in previous comment.